specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2018-08-01/ResourceHealth.json

```diff
@@ -1095,6 +1128,156 @@
         }
       }
     },
+    "statusBanner": {
+      "description": "Banner type of emerging issue.",
+      "properties": {
+        "title": {
+          "type": "string",
+          "description": "The banner title."
+        },
+        "message": {
+          "type": "string",
+          "description": "The details of banner."
+        },
+        "cloud": {
+          "type": "string",
+          "description": "The cloud type of this banner, eg. Public."
+        },
+        "lastModifiedTime": {
+          "type": "string",
+          "description": "The last time modified on this banner.",
+          "format": "date-time"
+        }
+      }
+    },
+    "impactedRegion": {
+      "description": "Object of impacted region.",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The impacted region slug id."
```

Comment:
I have no idea what a "slug ID" is and I work in Azure.  Will a user know?

Reply:
slug id is just a unique id we used in azure.com. I can remove it to be only 'id'.

Reply:
done.